src/System.Security.Cryptography.Xml/tests/Samples/EncryptingDecryptingSymmetricKeyWrap.cs

```diff
@@ -0,0 +1,113 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Xml;
+using Xunit;
+
+namespace System.Security.Cryptography.Xml.Tests
+{
+    public class EncryptingDecryptingSymmetricKeyWrap
+    {
+        private static XmlDocument LoadXmlFromString(string xml)
+        {
+            var doc = new XmlDocument();
+            doc.PreserveWhitespace = true;
+            doc.LoadXml(xml);
+            return doc;
+        }
+
+        private static void Encrypt(XmlDocument doc, string elementName, string encryptionElementID, SymmetricAlgorithm key, string keyName, Func<SymmetricAlgorithm> innerKeyCtor)
+        {
+            var elementToEncrypt = (XmlElement)doc.GetElementsByTagName(elementName)[0];
+
+            using (SymmetricAlgorithm innerKey = innerKeyCtor())
+            {
+                // Encrypt the key with another key
+                var encryptedKey = new EncryptedKey()
+                {
+                    CipherData = new CipherData(EncryptedXml.EncryptKey(innerKey.Key, key)),
+                    EncryptionMethod = new EncryptionMethod(TestHelpers.GetEncryptionMethodName(key, keyWrap: true))
```

Comment:
this also works when you don't use `keyWrap: true` but it is also not telling about keywrapping in the resulting xml (I don't believe additional spec URL makes sense here - IMO SignedXml should do non-wrapped to wrapped URL conversion here or the URL should be gone)